### DISCUSSION TITLE
System Cannot Find The Specified Path

### DISCUSSION THREAD

**Discussion by bali2126** [nextgenhealthcare/connect#6082] [Q&A]
**Describe the bug**
When I pull all the channels from mirth Connect Server using cmd with command and I get this error. How I can fixed that error.

E:\mirth\iehr-mirth\nabidh_prod\Channels\Endocare Clinic FZ LLC \index.xml (The system cannot find the path specified).

**Comment by jonbartels** [comment]
You need to show the command you are running.

**Comment by bali2126** [comment]
Hi

Following is Command that I am using to pull data from mirth.

java -jar mirthsync.jar -s https://10.0.0.0/api -u farhan -p  -i pull -t "path to environment folder"
________________________________

**Reply by pacmano1** [reply]
mirthsync is not a core function of mirth connect.   See https://github.com/SagaHealthcareIT/mirthsync.  Ask your question over there pehaps.

also, please do not open issues unless you have a confirmed bug. It is usually best to start a discussion first I have moved your issue to the discussions area.
